common/types/src/main/java/io/helidon/common/types/TypeInfo.java

```diff
@@ -1416,21 +1628,21 @@ protected TypeInfoImpl(TypeInfo.BuilderBase<?, ?> builder) {
                 this.typeName = builder.typeName().get();
                 this.rawType = builder.rawType().get();
                 this.declaredType = builder.declaredType().get();
-                this.description = builder.description();
+                this.description = builder.description().map(Function.identity());
```

Comment:
What does this do?

Reply:
This is code generated.

Reply:
It is needed if there are generics defined. I do not want to "clean-up" the generated code for `common-types`, as it is easier to check for changes when we need an update.